swap/tests/testutils/mod.rs

```diff
@@ -107,7 +106,12 @@ pub struct TestContext {
 }
 
 impl TestContext {
-    pub async fn new_swap_as_bob(&mut self) -> (bob::Swap, BobEventLoopJoinHandle) {
+    pub async fn alice_swap_triggered_by_bob(&mut self) -> alice::Swap {
```

Comment:
Maybe `next_swap` would be a better name together with a doc comment (`///`) that explains that in a test context, the next swap will be the one that Bob triggered because there will always be just one swap.